tests/operator/operator.go

```diff
@@ -1520,7 +1520,7 @@ spec:
 			checkVirtComponents(nil)
 
 			By("Starting a VMI")
-			vmi := tests.NewRandomVMI()
+			vmi := libvmi.New(libvmi.WithResourceMemory("1Mi"))
```

Comment:
Thanks for this PR. I think we can reduce the footprint of our tests quite significantly by running smaller VMs when appropriate.

The entire test begs to be converted to a unit test, as it does not communicate with qemu or the guest, only checking the rendered virt-launcher pod. But this is besides the point of this PR.

Reply:
+1.
Will do in a separate PR.

Reply:
Update: I re-examined the test, and I think this scenario cannot be easily created with unit tests. 
This tests indeed does not communicate with qemu or the guest, but it does communicate with the k8s RBAC server, making sure there are no secrets added to the virt-launcher pod.

Reply:
I don't know what you mean by RBAC server, or why it is relevant to this test. I may well be wrong, but this test just checks a certain property of the virt-launcher spec. This can be generated and verified without involving a running k8s system.